pennylane/wires.py

```diff
@@ -495,15 +539,15 @@ def unique_wires(list_of_wires):
         return Wires(tuple(unique), _override=True)
 
     def union(self, other):
-        """Return the union of the current Wires object and either another Wires object or an
-        iterable that can be interpreted like a Wires object e.g., List.
+        """Return the union of the current :class:`~.Wires` object and either another :class:`~.Wires` object or an
+        iterable that can be interpreted like a :class:`~.Wires` object, e.g., a ``list``.
 
         Args:
-            other (Any): Wires or any iterable that can be interpreted like a Wires object
-                to perform the union with. See _process for details on the interpretation.
+            other (Any): :class:`~.Wires` or any iterable that can be interpreted like a :class:`~.Wires` object
+                to perform the union with. See :func:`~.process` for details on the interpretation.
```

Comment:
I saw that @dwierichs was not so happy with referencing a private function. Probably can reference Wires in this instance as an alternative :)

Reply:
True, I was still missing to erase this reference in the methods. @albi3ro suggested to instead just reference the `Wires` constructor as you mention as well